uds/translator/data_record_definitions/formula.py

```
@@ -497,28 +497,28 @@ def get_coefficients_formula(scaling_byte_number: int) -> Callable[[int], Tuple[
+    :param subfunction: SecurityAccess SubFunction value.
 
     :return: Data Records that are present in the request message after given SubFunction value.
     """
-    if sub_function % 2:
+    if subfunction % 2:
         return (SECURITY_ACCESS_DATA,)
     return (SECURITY_KEY,)
 
 
-def get_security_access_response(sub_function: int) -> Union[Tuple[RawDataRecord], Tuple[()]]:
+def get_security_access_response(subfunction: int) -> Union[Tuple[RawDataRecord], Tuple[()]]:
```

Ruff (0.15.20)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)